erpnext/setup/install.py

```diff
@@ -66,6 +67,16 @@ def create_compact_item_print_custom_field():
 	})
 
 
+def create_print_uom_after_qty_custom_field():
+	create_custom_field('Print Settings', {
```

Comment:
WHY CUSTOM FIELD? WHY CAN'T YOU ADD THIS AS A STANDARD FIELD IN THE PRINT SETTINGS?

Reply:
I don't know how! Print Settings are in frappe/frappe, not frappe/erpnext. HOW CAN I DO THIS?

Reply:
And no, adding it in frappe/frappe is not a solution, because it will be visible for every frappe user, even if not using erpnext. I WANT to do it in correct way, but you didn't help and nobody in the forum help.

Reply:
Yeah makes sense, then instead of Print settings, can you add it in the sales/buying settings?

Reply:
I don't think so. It's for every document: buying, selling, stock and everything else what prints quantity.